# This service is used to retrieve an existing object from the interoperability
# server.

# Object ID.
uint64 id

---

# Whether the request was successful.
bool success

# The requested object.
Object object
